lib/JobType.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+class JobType
+{
+    const BIOMETRIC_KYC = 1;
+    const SMART_SELFIE_REGISTRATION = 2;
+    const SMART_SELFIE_AUTHENTICATION = 4;
```

Comment:
```suggestion
    const SMART_SELFIE_AUTHENTICATION = 2;
    const SMART_SELFIE_REGISTRATION = 4;
```
This is a bug in the other libs
https://docs.smileidentity.com/further-reading/faqs/how-do-job-types-map-to-the-new-product-names